src/wc.c

```diff
@@ -108,6 +120,29 @@ static struct option const longopts[] =
   {NULL, 0, NULL, 0}
 };
 
+#ifdef USE_AVX2_WC_LINECOUNT
+static bool
+avx2_supported(void)
```

Comment:
The detection logic is incorrect. It has to check `OSXSAVE` in (XCR0) as well.

Reply:
Aha, did not know that. Thanks!

Reply:
Yo can test the difference by booting with `noxsave`, by the way. This won't clear the CPUID bits, but the instructions will fault.

Reply:
Fixed now.